### DISCUSSION TITLE
Kinetic constants of an enzyme in reversible drug metabolism

### DISCUSSION THREAD

**Discussion by shapoo1** [Open-Systems-Pharmacology/Forum#846] [Questions & Problems] [answered]
I am trying to include a reversible drug metabolite reaction in PKsim (A ⇌ B). I have Vmax and Km value of enzyme for the forward reaction and only Km value for backward reactionwhich I obtained from literature. How can I optimized the desired Vmax value of the same enzyme in reversible reaction?

**Comment by StephanSchaller** [comment, marked as answer]
@shapoo1, you simply add another saturating (michaelis menten) metabolizing enzyme reaction for the metabolite with the parent as its metabolite. 
You can then fit the Vmax on observed PK data.
You might need to consider some kind of interaction between parent and metabolite on the metabolizing enzyme

best,
Stephan

**Comment by shapoo1** [comment]
Hi Stephan!
Thanks a lot for your reply. I have already set up a separate reaction, from metabolite to parent drug by adding metabolite as a separate compound and then in configuration process by selecting both drug as well as its metabolite in the enzymatic process. But the simulation results look like this:
Upper one is the parent drug Simulation and lower one is reversible reaction of metabolite

**Comment by StephanSchaller** [comment]
I cant see any simulated results / plots?

**Comment by shapoo1** [comment]
Simulation image

**Comment by shapoo1** [comment]
sorry I am not able to upload it
